homeassistant/components/binary_sensor/doorbird.py

```diff
@@ -36,7 +56,13 @@ def __init__(self, device, sensor_type):
         """Initialize a binary sensor on a DoorBird device."""
         self._device = device
         self._sensor_type = sensor_type
-        self._state = None
+        self._assume_off = datetime.datetime.now()
```

Comment:
If we don't get an actual off signal from the device, we should just publish this push as an event, rather than creating a binary sensor for it.